### ISSUE TITLE
CIL currently failing on Travis

### ISSUE THREAD

**Issue by rijobro** [SyneRBI/SIRF-SuperBuild#309]
PR #307 only changes a few lines of code, but many of the Travis builds are failing with errors along the lines of:
```
======================================================================
ERROR: test_DataProcessor (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_DataProcessor
Traceback (most recent call last):
  File "/usr/lib/python3.5/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib/python3.5/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/home/travis/build/CCPPETMR/SIRF-SuperBuild/sources/CCPi-Framework/Wrappers/Python/test/test_DataProcessor.py", line 21, in <module>
    from ccpi.framework import DataProcessor
  File "/home/travis/build/CCPPETMR/SIRF-SuperBuild/INSTALL/python/ccpi/framework/__init__.py", line 30, in <module>
    from .framework import DataContainer
  File "/home/travis/build/CCPPETMR/SIRF-SuperBuild/INSTALL/python/ccpi/framework/framework.py", line 47, in <module>
    cilacc = ctypes.cdll.LoadLibrary(dll)
  File "/usr/lib/python3.5/ctypes/__init__.py", line 425, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python3.5/ctypes/__init__.py", line 347, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libcilacc.so: cannot open shared object file: No such file or directory
```
I haven't done any digging, but I suspec this is due to an upstream change to CIL?

**Comment by rijobro** [comment]
PR #310 will temporarily disable Travis CIL until fixed.

**Comment by paskino** [comment]
Will be closed by #368
